netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -539,54 +544,71 @@ private void createStream(final CreateStreamCommand command, final ChannelPromis
       return;
     }
 
-    final NettyClientStream.TransportState stream = command.stream();
-    final Http2Headers headers = command.headers();
+    NettyClientStream.TransportState stream = command.stream();
+    Http2Headers headers = command.headers();
     stream.setId(streamId);
 
+    PerfMark.startTask("NettyClientHandler.createStream", stream.tag());
```

Comment:
not related to this PR, but returning task (or closable for startTask method) is maybe cleaner than passing string argument.

Reply:
Ack